README.md

```diff
@@ -60,6 +60,12 @@ The [Infracost GitHub action](https://github.com/marketplace/actions/run-infraco
 
 <img src="https://raw.githubusercontent.com/infracost/infracost-gh-action/master/screenshot.png" width=600 alt="Example infracost diff usage" />
 
+### Docker
```

Comment:
Maybe we should add a `Docker:` item after `Mac OSX:` above in the installation with `docker pull infracost/infracost` ?